package.json

```diff
@@ -40,6 +40,7 @@
     "@google-cloud/common": "^0.11.0",
     "continuation-local-storage": "^3.1.4",
     "gcp-metadata": "^0.1.0",
+    "is": "^3.2.0",
```

Comment:
I think it was used in errors. If `is` is the best way going forward we should switch things over there.